### PR TITLE
Add bulk actions explanation and main pages description

### PR DESCRIPTION
**Pull request by gsanchezu**
@robersantamaria, @SergioQCL, 👍?

### CODE REVIEW

File: Cloud Application Manager/Core Concepts/boxes.md [hunk 1]
```diff
@@ -37,17 +38,41 @@ Boxes contain scripts, variables, and metadata to automate processes when instan
...
+* [Script boxes](../Automating Deployments/script-box.md) - Automate using Bash, PowerShell, Salt, Ansible, Puppet, or Chef.
+* [Container boxes](../Automating Deployments/docker-container-service.md) - Automate using container technology like Docker.
+* [Template boxes](../Automating Deployments/template-box.md) - Automate using AWS CloudFormation templates, ARM templates or Terraform templates.
+* [Deployment Policy boxes](../Automating Deployments/deploymentpolicy-box.md) - Select and share infrastructure resources, networking, and more from a cloud provider.
+
+In the left side menu you can click on any of these types to filter by it on the view.
+
+### New Box
+
+To create a new box, click **New** button in the **Boxes** page. Select a box type to match your automation:
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/boxes.md]: ...click **on/the/on the** New button...

File: Cloud Application Manager/Core Concepts/providers.md [hunk 2]
```diff
@@ -31,7 +33,19 @@ An active Cloud Application Manager account.
...
 A provider is a public or private cloud account you register in Cloud Application Manager. Cloud Application Manager interfaces with the provider account’s API to provision and orchestrate deployments automatically. Before you can deploy workloads from Cloud Application Manager, you need to register a provider first.
 
-#### Supported provider types
+### Providers page
+
+When you click the **Providers** option in the left side menu you will see the providers list you have access to with some basic information for each provider such as its name, synchronization status, when it was last modified, the account alias and the owner who created it.
+
+![Providers list](../../images/cloud-application-manager/core-concepts/providers-list.png)
+
+You can create new providers by clicking on the **New** button, type something in the search field to search for a provider in the list or filter by the workspace they belong to.
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/providers.md]: ...to **seek** for...

File: Cloud Application Manager/Core Concepts/providers.md [hunk 3]
```diff
@@ -31,7 +33,19 @@ An active Cloud Application Manager account.
...
+
+Every provider in the list have a gear icon from which you can synchronize or delete the provider. If you click on any of the provider listed, you will get to the provider details page, where you can find more information and interact with it.
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/providers.md]: ...any of the provider**s** listed, you will get to **its** details page...

File: Cloud Application Manager/Core Concepts/providers.md [hunk 4]
```diff
@@ -31,7 +33,19 @@ An active Cloud Application Manager account.
...
+
+If you select several providers by clicking on the check box on the provider line item (or clicking the check box in the header line which will select all), the **Bulk Actions** button dropdown will appear next to the *New* provider button, allowing to sync or delete the selected providers in bulk mode.
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/providers.md]: I think checkbox is more widely used than check box. Do you agree?

File: Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md [hunk 5]
```diff
@@ -251,54 +252,62 @@ Go to the Admin Console to [manage several instances](../Administering Your Orga
...
 
 This shows the instance’s details that you can modify. These details are: icon, name, description, tags, expiration, updates (available for box versions only) and automatic reconfigure.
 
+### Managing multiple instances with bulk actions
+
+Many of the instance actions can be applied in bulk to several instances at once.
+
+![Bulk actions menu](../../images/cloud-application-manager/deploying-anywhere/bulk-actions.png)
+
+The **Bulk Actions** button dropdown appears next to the *New* instance button once you select any instance in the list and, depending on your selection, the common actions allowed in all selected instances will be available in the dropdown. For example, if you select a few instances all in *Online* state, you will have all the applicable lifecycle actions available (**Reinstall**, **Reconfigure**, **Shutdown**, **Terminate**, **Force Terminate**) and any other additional option such as *Update Instance*, but if you also include an *Unavailable* instance in your selection among the *Online* ones, the only available lifecycle option will be **Force Terminate**.
```
Comment (SergioQCL) [inline Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md]: ...a few (Online) instances.../...a few instances (comma) all **showing an** *Online* state...

...such as *Update Instance* (semi-colon) but if...

File: Cloud Application Manager/Core Concepts/providers.md (hunk 2 above)
Comment (gsanchezu) [reply Cloud Application Manager/Core Concepts/providers.md]:
```suggestion
You can create new providers by clicking on the **New** button, type something in the search field to look for a provider in the list or filter by the workspace they belong to.
```

**Comment by gsanchezu** [comment]
I've addressed all your comments @SergioQCL, thanks!

File: Cloud Application Manager/Core Concepts/boxes.md [hunk 6]
```diff
@@ -37,17 +38,41 @@ Boxes contain scripts, variables, and metadata to automate processes when instan
...
-* [Template](../Automating Deployments/template-box.md) - Automate using AWS CloudFormation templates, ARM templates or Terraform templates.
-* [Deployment Policy](../Automating Deployments/deploymentpolicy-box.md) - Select and share infrastructure resources, networking, and more from a cloud provider.
+The **tiles view** displays a big icon, name, type and the owner of the box. When you hover on a tile, two action buttons appear allowing you to deploy or manage the box. You can sort the tiles by name (default), last modified or owner by selecting one of these values in the *Sort by* dropdown located on the top-right corner of the page, next to the change view mode button.
+
+![Box page in list view](../../images/cloud-application-manager/core-concepts/boxes-list.png)
+
+The **list view** displays for each element a small icon, name, type, when it was last modified, the owner and two action buttons for each box. You can order by any column by clickin on its header and f you select any of the boxes in the list, a *Bulk actions* button appears, so that selected boxes can be updated all at once.
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/boxes.md]: That 'g' is still missing, and so is the 'i' from if. Apart from that, I think the point you had after _header_ makes sense here to separate the sentence. Let's discuss it if you want ;)

File: Cloud Application Manager/Core Concepts/boxes.md [hunk 7]
```diff
@@ -37,17 +38,41 @@ Boxes contain scripts, variables, and metadata to automate processes when instan
...
+
+The *gear icon* action button allows you to manage the box by performing additional actions such as sharing the box, editing its details, cloning or deleting it. These options will be available depending on your access permissions onto the box.
+
+You can also use the *filters* above to narrow down the boxes being displayed by the workspace they belong to or the tags they have associated.
+
+There are several types of boxes to address for your specific automation use case:
+
+* [Application boxes](../Automating Deployments/application-box.md) allows to configure several boxes to deploy an application with a single click.
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/boxes.md]: I doubt here: should we use singular (regarding a type) or plural (regarding boxes)? In both cases some verbs (don't) need an 's'

File: Cloud Application Manager/Core Concepts/providers.md [hunk 8]
```diff
@@ -31,7 +33,19 @@ An active Cloud Application Manager account.
...
-#### Supported provider types
+### Providers page
+
+When you click the **Providers** option in the left side menu you will see the providers list you have access to with some basic information for each provider such as its name, synchronization status, when it was last modified, the account alias and the owner who created it.
+
+![Providers list](../../images/cloud-application-manager/core-concepts/providers-list.png)
+
+You can create new providers by clicking on the **New** button, type something in the search field to look for a provider in the list or filter by the workspace they belong to.
+
+Every provider in the list have a gear icon from which you can synchronize or delete the provider. If you click on any of the provider listed, you will get to its details page, where you can find more information and interact with it.
```
Comment (SergioQCL) [inline Cloud Application Manager/Core Concepts/providers.md]: ...provider**s** listed...

**Pull request merged**